Detectors/EMCAL/base/include/EMCALBase/EMCGeometry.h

```diff
@@ -0,0 +1,378 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_EMCAL_EMCGEOMETRY_H_
+#define ALICEO2_EMCAL_EMCGEOMETRY_H_
+
+#include <iosfwd>
+
+#include <TArrayD.h>
+#include <TList.h>
+#include <TMath.h>
+#include <TNamed.h>
+#include <TString.h>
+
+#include <FairLogger.h>
+
+class TObjArray;
+
+namespace o2
+{
+namespace EMCAL
+{
+class EMCGeometry : public TNamed
```

Comment:
No, but this will be fixed in a later stage. During the coding sprint we will have no time. However it is clearly stated that the code needs cleanup.

Maybe can be converted in an issue forcing this to be followed up.